vital/exceptions/base.cxx

```diff
@@ -64,21 +64,21 @@ ::set_location( std::string const& file, int line )
 
 // ------------------------------------------------------------------
 char const*
-vital_core_base_exception
+vital_exception
 ::what() const noexcept
 {
   return this->m_what.c_str();
 }
 
 
-// ------------------------------------------------------------------
+// ============================================================================
 invalid_value
 ::invalid_value(std::string reason) noexcept
-  : m_reason(reason)
 {
   m_what = "Invalid value(s): " + reason;
 }
 
+
```

Comment:
One of these days we should sit down to agree on "official" style rules, but FWIW, my preference is for separator lines always (present above, but missing here) and *never* consecutive blank lines. (Use of separator lines makes consecutive blank lines unnecessary, and "never" is a very easy to remember rule about when to use consecutive blank lines.)

Reply:
.... and while we're on it, my preference for separator lines has always been that the end on column 79 as a clue that you are exceeding reasonable limits of a line of code and might consider adding a new line.